### PR TITLE
Correct dropwizard dependencies

### PR DESCRIPTION
**Pull request by mosabua**
Part of #184

### CODE REVIEW

File: gateway-ha/pom.xml [hunk 1]
```diff
@@ -81,30 +81,53 @@
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-assets</artifactId>
         </dependency>
-
         <dependency>
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-auth</artifactId>
         </dependency>
-
+        <dependency>
```
Comment (ebyhr) [inline gateway-ha/pom.xml:88]: Please add empty lines.

**Review by ebyhr — APPROVED** [review]

File: gateway-ha/pom.xml (hunk 1 above)
Comment (mosabua) [reply gateway-ha/pom.xml:88]: Is this the trino style even for dependencies that belong together? Its soooo verbose

Comment (ebyhr) [reply gateway-ha/pom.xml:88]: That's the style in Trino repository.

Comment (mosabua) [reply gateway-ha/pom.xml:88]: ok.. will update

**Pull request merged**